src/components/MainNav/AppList/AppList.js

```diff
@@ -31,6 +32,9 @@ class AppList extends Component {
     dropdownId: PropTypes.string,
     dropdownToggleId: PropTypes.string.isRequired,
     selectedApp: PropTypes.object,
+    stripes: PropTypes.shape({
+      locale: PropTypes.string.isRequired,
```

Comment:
We could avoid having to pass around the `stripes` god object here by referring to `locale` from `react-intl` instead. I did a similar thing with `<Datepicker>` where I switched to using `intl.locale` instead of `stripes.locale` (or worse, `stripes.intl.locale`).

Maybe could make use of the new `<IntlConsumer>`? https://github.com/folio-org/stripes-core/pull/535

Reply:
No problem! I'll make that change 👍

Reply:
Good idea! We can apply `<IntlConsumer>` in `MainNav.js` and use it to pass `intl` to `<AppList>` as a prop.